### PR TITLE
Dala 2323 view page show linked documents

### PR DESCRIPTION
**Pull request by pioSko2**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed
  - [ ] The main branch is deployed to a dev server with `Reset non-user related Docker Volumes & Re-populate` turned on
  - [ ] It's verified that the CD run is green
  - [ ] Immediately after that the feature branch is deployed to the same server with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the dev server
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue [hunk 1]
```diff
@@ -3,7 +3,17 @@
...
     <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
   </div>
-  <div v-show="!waitingForData">
+  <div v-show="!waitingForData" data-test="multipleReportsBanner">
+    <ShowMultipleReportsBanner
+      data-test="multipleReportsBanner"
+      v-if="
+        extractedReportsAndReportingPeriods &&
+        extractedReportsAndReportingPeriods[0] &&
+        extractedReportsAndReportingPeriods[1]
```
Comment (pioSko2) [inline dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue:12]: Why are we specifically checking for [0] and [1] ? Couldn't we check `extractedReportsAndReportingPeriods?.length > 1` ?

File: dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue [hunk 2]
```diff
@@ -176,6 +202,30 @@ export default defineComponent({
...
+    extractReportsAndReportingPeriodsFromDataAndMetaInfoSets(
+      dataAndMetaInfoSets: Array<DataAndMetaInformationEuTaxonomyDataForNonFinancials>,
+    ): [({ [p: string]: CompanyReport } | undefined)[], Array<string>] {
+      const reportingPeriods = [];
+      let tempReportingPeriod: string | undefined;
+      for (const dataAndMetaInfoSet of dataAndMetaInfoSets) {
+        tempReportingPeriod = dataAndMetaInfoSet.metaInfo.reportingPeriod;
+        if (tempReportingPeriod) {
+          reportingPeriods.push(tempReportingPeriod);
+        } else console.log("no reporting period given");
```
Comment (pioSko2) [inline dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue:221]: This `else` should be removed. We don't need console.logs.

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/euTaxoNonFinancials/EuTaxonomyForNonFinancialsPanel.cy.ts [hunk 3]
```diff
@@ -80,4 +83,74 @@ describe("Component test for the NewEUTaxonomy Page", () => {
...
+      {
+        keycloak: minimalKeycloakMock({}),
+      },
+      {
+        reports: reportsAndReportingPeriods[0],
+        reportingPeriods: reportsAndReportingPeriods[1],
+      },
+    ).then(() => {
+      cy.get(`[data-test="frameworkNewDataTableTitle"`).contains(
+        "Data extracted from the company report.Company Reports",
```
Comment (pioSko2) [inline dataland-frontend/tests/component/components/resources/frameworkDataSearch/euTaxoNonFinancials/EuTaxonomyForNonFinancialsPanel.cy.ts:100]: Space after `report. ` .. Should be ` report. Company `

File: dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue (hunk 2 above)
Comment (marcelrasbachdfine) [reply dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue]: done

File: dataland-frontend/tests/component/components/resources/frameworkDataSearch/euTaxoNonFinancials/EuTaxonomyForNonFinancialsPanel.cy.ts (hunk 3 above)
Comment (marcelrasbachdfine) [reply dataland-frontend/tests/component/components/resources/frameworkDataSearch/euTaxoNonFinancials/EuTaxonomyForNonFinancialsPanel.cy.ts]: done

File: dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue (hunk 1 above)
Comment (marcelrasbachdfine) [reply dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue:12]: also, by definition extractedReportsAndReportingPeriods has the length of 2, regardless of both entries.

**Pull request merged**